src/Ens/JobeetBundle/Tests/Controller/DefaultControllerTest.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Ens\JobeetBundle\Tests\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+
+class DefaultControllerTest extends WebTestCase
+{
+    public function testIndex()
+    {
+        $client = static::createClient();
+
+        $crawler = $client->request('GET', '/hello/Fabien');
+
+        $this->assertTrue($crawler->filter('html:contains("Hello Fabien")')->count() > 0);
```

Comment:
@PhotonSapiens 
PHPUnit considers method as test if it has `@test` annotation **OR** starts with test. We use first method